handler.py

```diff
@@ -14,38 +16,120 @@
 SQS_CLIENT = boto3.client('sqs')
 
 
-def runScheduledObservatoryScan(event, context):
+def addPortScanToQueue(event, context):
+    data = json.loads(event['body'])
+    target = Target(data.get('target'))
+
+    if target.isValid():
+        print(SQS_CLIENT.send_message(
+              QueueUrl=os.getenv('SQS_URL'),
+              MessageBody="portscan|" + target.targetname
+              ))
+
+        return Response({
+            "statusCode": 200,
+            "body": json.dumps({'OK': 'target added to the queue'})
+        }).with_security_headers()
+
+    else:
+        logger.error("Target Validation Failed of: " +
+                     json.dumps(target.targetname))
+        return Response({
+             "statusCode": 400,
+             "body": json.dumps({'error': 'target was not valid or missing'})
```

Comment:
You use all caps in the positive response to queuing, but use all lower for an error condition.  Should we consider a consistent cap scheme here?

Reply:
Sure, will update.